Gruntfile.js

```diff
@@ -176,33 +180,53 @@ module.exports = function( grunt ) {
 	} );
 
 	// Load grunt tasks from NPM packages
-	require( "load-grunt-tasks" )( grunt );
+	// Support: Node.js <4
+	// Don't load the eslint task in old Node.js, it won't parse.
```

Comment:
Actually, it's not `chalk` but `grunt-eslint`:
https://github.com/sindresorhus/grunt-eslint/commit/d0ff94266c668a5d6ad2bfd5bb0e92445f6a580a